README.md

```diff
@@ -9,11 +9,15 @@ This project shows how Fable can be used fullstack with a node.js server written
 - [Mono Framework](https://www.mono-project.com/download/stable/) for some tooling if working in non-Windows environment
 - An F# IDE, like Visual Studio Code with [Ionide extension](http://ionide.io/)
 
-## Installing dependencies
+## Installing & running (in dev mode)
 
-Type `npm install` to install dependencies (for both JS and F#) after cloning the repository or whenever dependencies change.
+- `.\.paket\paket.bootstrapper.exe`
+- `.\.paket\paket.exe update`
```

Comment:
This will update dependencies and change the `paket.lock` file, possibly breaking the build.